marklogic-data-hub-central/ui/src/assets/mock-data/flows.data.ts

```diff
@@ -473,6 +474,18 @@ const loadSettings = {"data" :
     "status" :200
 };
 
+const customStepsWithEntity = {"data" :[{
+        "entityType": "Customer",
+        "entityTypeId": "http://example.org/Customer-0.0.1/Customer",
+        "artifacts": [{...commonData.customData[0]}]
+    }],
+    "status" :200
+}
+
+const customStepsWithoutEntity = {"data" :[{...commonData.customData[1]}],
```

Comment:
Should this be customData[0]?

Reply:
No, it should be customData[1] as it is not associated with any entity